### PR TITLE
connectors: added managed connectors to shared responsibility model

### PR DESCRIPTION
**Pull request by bochenekmartin**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/<add-your-issue-number-here>
Review deadline:

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/get-started/pages/cloud-overview.adoc [hunk 1]
```diff
@@ -165,7 +169,9 @@ BYOC with customer-managed VPC::
 | Proactive threat detection | &#10003; | &#10003;  
 | Static secret rotation | &#10003; | &#10003;  
 | Incident response | &#10003; |   
-| Resilience verification | &#10003; | 
+| Resilience verification | &#10003; |
+| Managed Connectors infrastructure | &#10003; | &#10003;
+| Managed Connectors tasks state  |  | &#10003; |
```
Comment (micheleRP) [inline modules/get-started/pages/cloud-overview.adoc:174]: @bochenekmartin I think as soon as we support Redpanda Connect in Cloud, we'll rename "Managed Connectors" to "Kafka Connect". Should we do that with this table update?

Comment (bochenekmartin) [reply modules/get-started/pages/cloud-overview.adoc:174]: @micheleRP, yes, good idea!

File: modules/get-started/pages/cloud-overview.adoc [hunk 2]
```diff
@@ -105,7 +105,9 @@ Dedicated::
 | Proactive threat detection | &#10003; |  
 | Static secret rotation | &#10003; |  
 | Incident response | &#10003; |   
-| Resilience verification  | &#10003; |  
+| Resilience verification  | &#10003; |
+| Managed Connectors infrastructure  | &#10003; |
+| Managed Connectors tasks state  |  | &#10003;
```
Comment (micheleRP) [inline modules/get-started/pages/cloud-overview.adoc:110]:
```suggestion
| Kafka Connect infrastructure  | &#10003; |
| Kfka Connect tasks state  |  | &#10003;
```

File: modules/get-started/pages/cloud-overview.adoc [hunk 3]
```diff
@@ -135,7 +137,9 @@ BYOC::
 | Proactive threat detection | &#10003; | &#10003;  
 | Static secret rotation | &#10003; |  
 | Incident response | &#10003; |     
-| Resilience verification  | &#10003; |  
+| Resilience verification  | &#10003; |
+| Managed Connectors infrastructure | &#10003; | &#10003;
+| Managed Connectors tasks state  |  | &#10003; |
```
Comment (micheleRP) [inline modules/get-started/pages/cloud-overview.adoc:142]:
```suggestion
| Kafka Connect infrastructure | &#10003; | &#10003;
| Kafka Connect tasks state  |  | &#10003; |
```

File: modules/get-started/pages/cloud-overview.adoc (hunk 1 above)
Comment (micheleRP) [inline modules/get-started/pages/cloud-overview.adoc:174]:
```suggestion
| Kafka Connect infrastructure | &#10003; | &#10003;
| Kafka Connect tasks state  |  | &#10003; |
```

**Review by micheleRP — APPROVED** [review]
Thanks @bochenekmartin!

**Pull request merged**